CHANGELOG.md

```diff
@@ -1,6 +1,44 @@
 # Changelog
 
-All notable changes to this project will be documented in this file.
+All notable changes to this project will be documented in this file
+
+## v20.11.0
+
+### Added
+
+* Activate SAML when using single sign-on (SSO)
+* Add a route in OpenShift to allow connections to SSO, when enabled
+* Updated the `ServerEAR.Dockerfile` to reduce layers
+* Added note with fix needed for an update in IBM MQ, the details of which can be found
+[here](https://github.com/IBM/charts/blob/master/stable/ibm-mqadvanced-server-dev/RELEASENOTES.md).
+* Added links to Architecture and Troubleshooting sections from within the flow of the document
+
+### Changed
+
+* Updated SPM 7.0.10.0 Supported Prereqs.
+  * Kubernetes version 1.18 supported introduced.
```

Comment:
"Kubernetes version 1.18 supported introduced"
->
"Kubernetes version 1.18 support introduced" or "Kubernetes version 1.18 introduced"

Reply:
What about _Introducing support for Kubernetes version 1.18_ ?

Reply:
👍